protos/class.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+import "protos/user.proto";
+import "protos/chat.proto";
+import "protos/error.proto";
+import "protos/quiz.proto";
+
+service PresenterClassService {
```

Comment:
musimy sie zdecydować na jedna konwencję nazw, jesli zostajemy przy nazewnictwie *Service to trzeba zaktualizować pozostałe pliki

Reply:
według mnie o ile tutaj jest to czytelne (bo wiadomo, że to serwis), ale w kodzie to lepiej by się czytało z tym przyrostkiem Service